### DISCUSSION TITLE
Weblate remote custom header authentication

### DISCUSSION THREAD

**Discussion by delenikov** [WeblateOrg/weblate#12876] [Q&A]
Hi everyone,

I've reviewed the Weblate documentation and didn’t find any specific information regarding support for remote authentication headers (custom headers), similar to what Django provides for remote user authentication (Django Remote User Auth Documentation). https://docs.djangoproject.com/en/5.1/howto/auth-remote-user/

Could you clarify if Weblate supports this type of authentication?

If this feature is supported, could you provide guidance, example or documentation on how can I implement it?

Thanks for your time and assistance

**Comment by nijel** [comment]
There is no reason why authentication itself would not work. You will end up with some integration issues (like sign out not working) as this has AFAIK never been tested.
